buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -1,28 +1,28 @@
 object Versions {
-    const val KOTLIN = "1.7.20"
+    const val KOTLIN = "1.8.10"
     const val ATOMIC_FU = "0.18.5"
```

Comment:
[Atomixfu 0.20.0](https://github.com/Kotlin/kotlinx-atomicfu/releases/tag/0.20.0) is out with Kotlin 1.8.10 support 🙂